easyrsa3/easyrsa

```diff
@@ -853,9 +853,18 @@ $(display_dn req "$req_in")
 		# add one to the extensions file
 		if [ "$crt_type" = 'server' ] || [ "$crt_type" = 'serverClient' ];
 		then
-			echo "$EASYRSA_EXTRA_EXTS" |
-				grep -q subjectAltName ||
-				default_server_san "$req_in"
+			echo "$EASYRSA_EXTRA_EXTS" | grep -q subjectAltName
```

Comment:
Client certificates may also have subject alternative names, e.g. e-mail addresses. Thus this line should be moved outside of the if-then-fi body.